jmclient/test/test_bond_calc.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+import pytest
+from jmclient import jm_single, load_test_config, FidelityBondMixin
+from jmclient.bond_calc import add_one_month, get_bond_values
+
+
+@pytest.mark.parametrize(('original_date', 'date_plus_one_month'),
+                         ((datetime(2022, 1, 1), datetime(2022, 2, 1)),
+                          (datetime(2022, 11, 1), datetime(2022, 12, 1)),
+                          (datetime(2022, 12, 1), datetime(2023, 1, 1))))
+def test_add_one_month(original_date: datetime, date_plus_one_month: datetime) -> None:
+    assert add_one_month(original_date) == date_plus_one_month
+
+
+def test_get_bond_values() -> None:
+    load_test_config()
+    # 1 BTC
+    amount = pow(10, 8)
+    months = 1
+    interest = 0.015
+    exponent = 1.3
+    jm_single().config.set("POLICY", "interest_rate", str(interest))
+    jm_single().config.set("POLICY", "bond_value_exponent", str(exponent))
```

Comment:
Same problem as was mentioned in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1254/files#r890587671

We address this same point in a bunch of other tests by a teardown/cleanup function, or you can do it directly in the test function etc.